course_catalog/api.py

```diff
@@ -117,10 +117,10 @@ def parse_mitx_json_data(course_data, force_overwrite=False):
                     # we need to find the newest changes
                     max_modified = max(course_modified, course_run_modified)
 
-                    # Try and get the CourseRun instance. If it exists check to see if it needs updating
+                    # Try and get the LearningResourceRun instance. If it exists check to see if it needs updating
                     try:
-                        courserun_instance = course.course_runs.get(
-                            course_run_id=course_run.get("key")
+                        courserun_instance = course.runs.get(
```

Comment:
should we rename this away from `courserun_instance`?

Reply:
or maybe `run_instance`? idk, maybe it's not important

Reply:
It actually is a run for a course here, so I left it as is.